tests/unit/backend/corpora/common/entities/test_dataset.py

```diff
@@ -124,11 +123,21 @@ def test__create_ids__ok(self):
         self.assertIsNotNone(dataset)
         self.assertEqual(dataset_id, dataset.id)
 
-        self.assertEqual(artifact_ids, get_ids(dataset.artifacts))
-        self.assertNotEqual(["test_dataset_artifact_id"], get_ids(dataset.artifacts))
+        self.assertEqual(artifact_ids, [i.id for i in dataset.artifacts])
+        self.assertNotEqual(["test_dataset_artifact_id"], [i.id for i in dataset.artifacts])
 
-        self.assertEqual(deployment_directory_ids, get_ids(dataset.deployment_directories))
-        self.assertNotEqual(["test_dataset_artifact_id"], get_ids(dataset.deployment_directories))
+        self.assertEqual(deployment_directory_ids, [i.id for i in dataset.deployment_directories])
+        self.assertNotEqual(["test_dataset_artifact_id"], [i.id for i in dataset.deployment_directories])
 
-        self.assertEqual(contributor_ids, get_ids(dataset.contributors))
-        self.assertEqual(["test_contributor_id"], get_ids(dataset.contributors))
+        self.assertEqual(contributor_ids, [i.id for i in dataset.contributors])
+        self.assertEqual(["test_contributor_id"], [i.id for i in dataset.contributors])
+
+    def test__list__ok(self):
+        generate = 5
+
+        for _ in range(generate):
+            Dataset.create(**DatasetParams.get())
+
+        datasets = Dataset.list()
+        self.assertGreaterEqual(len(datasets), generate)
```

Comment:
Likewise, please validate the exact ids you are expecting rather than the approximate length.

Reply:
fixed